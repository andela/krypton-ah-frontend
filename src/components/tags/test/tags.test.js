import React from 'react';
import { shallow } from 'enzyme';
import Tag from '../tags';

describe('button test', () => {
  it('should test for the button component', () => {
    const wrapper = shallow(<Tag />);
    expect(wrapper).toMatchSnapshot();
  });
});
