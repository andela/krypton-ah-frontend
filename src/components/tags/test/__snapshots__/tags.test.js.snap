// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test should test for the button component 1`] = `<componentloaderHOC />`;
